records.py

```diff
@@ -0,0 +1,29 @@
+from enum import Enum
+
+
+class FileEncoding(Enum):
```

Comment:
these enums seem relevant only to Files or Folders. why are we storing these outside of the scope of these?